content/docs/capabilities/enterprise-api.mdx

```diff
@@ -23,16 +22,22 @@ This doc assumes:
 
 ## Configure a new route
 
-1. We suggest configuring the route for API access in the open-source Pomerium. That way changes made through the API that might break access to the console GUI will not break access to the API route.
+1. We suggest configuring the route for API access in the open-source Pomerium Core. That way, changes made through the API that might break access to the Enterprise Console won't break access to the API route.
 
    ```yaml
    - from: https://console-api.localhost.pomerium.io
      to: https://pomerium-console-domain-name:8702
      pass_identity_headers: true
      allow_any_authenticated_user: true
-     tls_custom_ca_file: /path/to/rootCA.pem # See https://www.pomerium.com/reference/#tls-custom-certificate-authority
+     tls_custom_ca_file: /path/to/rootCA.pem
    ```
 
+   :::info
+
+   See [**TLS Custom Certificate Authority**](/docs/reference/routes/tls#tls-custom-certificate-authority) for more information about this route setting.
```

Comment:
I think it might be good to be explicit about what "this route setting" refers to.

```suggestion
   See [**TLS Custom Certificate Authority**](/docs/reference/routes/tls#tls-custom-certificate-authority) for more information about the `tls_custom_ca_file` setting.
```